tests/test_ain_account.py

```diff
@@ -0,0 +1,21 @@
+from unittest import TestCase
+from ain.account import Account
+from ain.utils import *
+from .data import (
+    mnemonic,
+    mnemonicPrivateKey,
+    mnemonicPublicKey,
+    mnemonicAddress,
+)
+
+class TestAccount(TestCase):
+    def testAccountFromMnemonic(self):
+        account = Account.fromMnemonic(mnemonic)
+        self.assertEqual(account.private_key, mnemonicPrivateKey.hex())
+        self.assertEqual(account.public_key, mnemonicPublicKey.hex())
+        self.assertEqual(account.address, mnemonicAddress)
+    
+    def testAccountCreateFromEntropyNone(self):
```

Comment:
could you add a test case with entropy != None?

Reply:
Added!